xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -190,9 +190,7 @@
                   "Application Insights Instrumentation Key": "[reference(concat('Microsoft.Resources/deployments/', 'sitecore-infra'), '2015-11-01').outputs.appInsightsInstrumentationKey.value]",
                   "Application Insights Role": "CM",
                   "KeepAlive Url": "[concat('https://', reference(resourceId('Microsoft.Web/sites', variables('cmWebAppNameTidy'))).hostNames[0], '/sitecore/service/keepalive.aspx')]",
-                  "License Xml": "[variables('licenseXml')]",
-                  "IP Security Client IP": "[parameters('security_clientIp')]",
```

Comment:
I only pulled them because the U3 XP0 CM package no longer had that IP Security Client IP Mask setting and it caused errors when it was passed (that the webdeploy package did not define it).